students/jemielity_kamil/lesson_07_string_datetime/table_printer.py

```diff
@@ -0,0 +1,22 @@
+def print_table(data):
+    col_width = []
+    for i in range(len(data)):
+        maximum = 0
+        for j in range(len(data[0])):
```

Comment:
More pythonic way is to go over items in the list instead of iterating over elements in the list

```
    for row in data:
        maximum = 0
        for item in row:
            if len(item) > maximum:
                maximum = len(item)
        col_width.append(maximum)
```

Or by using list comprehension:
```
max_lengths = [max(len(item) for item in column) for column in data]
```
Even shorten solution with map:
```
max_lengths = [max(map(len, column)) for column in data]
```